DigitalLearningSolutions.Data/Models/Objective.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace DigitalLearningSolutions.Data.Models
+{
+    using System.Collections.Generic;
+    using System.Linq;
+
+    public class Objective
+    {
+        public Objective(int tutorialId, string interactions, int possible)

```

Comment:
Taking in a string of interactions is very db-specific. Can you use a type handler instead? https://sodocumentation.net/dapper/topic/6/type-handlers